diff -up freetype-2.10.0/modules.cfg.2~ freetype-2.10.0/modules.cfg
--- freetype-2.10.0/modules.cfg.2~	2019-03-15 12:52:00.781635550 +0100
+++ freetype-2.10.0/modules.cfg	2019-03-15 12:54:09.606353195 +0100
@@ -111,7 +111,7 @@ AUX_MODULES += cache
 
 # TrueType GX/AAT table validation.  Needs `ftgxval.c' below.
 #
-# AUX_MODULES += gxvalid
+AUX_MODULES += gxvalid
 
 # Support for streams compressed with gzip (files with suffix .gz).
 #
@@ -130,7 +130,7 @@ AUX_MODULES += bzip2
 
 # OpenType table validation.  Needs `ftotval.c' below.
 #
-# AUX_MODULES += otvalid
+AUX_MODULES += otvalid
 
 # Auxiliary PostScript driver component to share common code.
 #
